### ISSUE TITLE
Laravel 5.8 default migrate has foreign key error

### ISSUE THREAD

**Issue by bolechen** [Silvanite/brandenburg#18]
In a laravel 5.8 fresh install, run `php artisan migrate` have error below:

```sql
   Illuminate\Database\QueryException  : SQLSTATE[HY000]: General error: 1215 Cannot add foreign key constraint (SQL: alter table `role_user` add constraint `role_user_user_id_foreign` foreign key (`user_id`) references `users` (`id`) on delete cascade)
```

because laravel 5.8 default change the id key to  `BIGINT` from `INT`

**Comment by m2de** [comment]
This should be resolved in the latest release 1.1.0

**Comment by florinmatthew** [comment]
Sorry, but this issue is not resolved in 1.1.0 version :(
`"laravel/framework": "5.7.*",`

**Comment by dam62500** [comment]
same problem in 1.2.0 -> 
~~~json
"laravel/framework": "5.8.*",
~~~
